### DISCUSSION TITLE
Query search on two fields by priority to one field's results

### DISCUSSION THREAD

**Discussion by mukarramali** [prisma/prisma#12667] [Q&A]
I have two columns called `title` and `description`. With raw queries, it looks like:

`SELECT * FROM tablename where (title like '%query%') or (description like '%query%') order by case
when title like '%query%' then 0
when description like '%query%' then 1
end;`

How can we prioritise search based on columns in Prisma?

**Comment by austincrim** [comment]
Hey @mukarramali 👋 ,

This is not possible in Prisma without using [`$rawQuery`](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#queryraw). It would be great if you could open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) outlining your use case.

Thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
